<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_encrypt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryptor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_decryptor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decryptor</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Chat_page">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>571</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="send_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>310</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Close Connection</string>
      </property>
     </widget>
     <widget class="QLabel" name="con_to_label">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="token_label">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>10</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Home">
     <widget class="QPushButton" name="connect_ip_button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_server_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Or</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ip_input">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port_input">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>250</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter port</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EncryptorWindow">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Message Encryption :-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Access Code :- </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="En_Input_msg">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_en_msg">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt Message</string>
      </property>
     </widget>
     <widget class="QLabel" name="Access_code">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Output :- </string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>471</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DecryptorWindow">
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Message Decryption :-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>444</x>
        <y>30</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt Message</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Access Code :-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>10</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Output :-</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>431</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Error_msg">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>359</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
